manifests/ruby.yml

```diff
@@ -567,6 +567,7 @@ tests/:
     test_remote_config_rule_changes.py:
       Test_AsmDdMultiConfiguration: missing_feature
       Test_BlockingActionChangesWithRemoteConfig: missing_feature (the test relies on remote AppSec activation)
+      Test_Unknown_Action: missing_feature (the test relies on remote AppSec activation)
```

Comment:
can we make the test not rely on remote activation? we need to validate Ruby too

Reply:
I'm actually not sure the test really relies on remote activation. It doesn't seem like it when I read the test case

Reply:
I did that because all the test cases in the test are set to missing_feature

```
    test_remote_config_rule_changes.py:
      Test_AsmDdMultiConfiguration: missing_feature
      Test_BlockingActionChangesWithRemoteConfig: missing_feature (the test relies on remote AppSec activation)
      Test_Unknown_Action: missing_feature (the test relies on remote AppSec activation)
      Test_UpdateRuleFileWithRemoteConfig: missing_feature (the test relies on remote AppSec activation)
```

The new test class is based on `Test_BlockingActionChangesWithRemoteConfig`, That’s why I assumed it would be the same in this case.